### PR TITLE
Add support for wildcard characters for archiving #122

### LINKED ISSUE #122: Support wildcard characters or some more regular expressions
That is request typical archivators, feature, like wildcard. `(*.*, *.ext, name.*)`

When i need use archiver for packing some format (only pdf, or something else), i should use bulkiness  `ioutil.ReadDir` + `filepath.Ext` or` filepath.Walk` +`filepath.Ext` metods
Maybe somebody else wants that feature in projekt?

### PR DESCRIPTION
**Pull request by Inconnu08**
Resolves #122

### CODE REVIEW

**Comment by Inconnu08** [comment]
@mholt what do you think?

File: cmd/arc/main.go [hunk 1]
```diff
@@ -58,7 +59,14 @@ func main() {
 		if !ok {
 			fatalf("the archive command does not support the %s format", iface)
 		}
-		err = a.Archive(flag.Args()[2:], flag.Arg(1))
+		sources := flag.Args()[2:]
+		if len(flag.Args()) == 3 {
```
Comment (mholt) [inline cmd/arc/main.go]: There could be more than 1 source, i.e. `archive dest.zip source1 source2*`

**Comment by Inconnu08** [comment]
@mholt you're welcome. This does the trick.

**Comment by mholt** [comment]
Great, thank you!

**Comment by Inconnu08** [comment]
My pleasure!

**Pull request merged**